src/pages/about/book.md

```diff
@@ -1,333 +1,30 @@
-
-Authors Daniel and Lucas, the architects behind Tauri take you on a journey from theory to execution, during which you will learn why Tauri was built and how it works under the hood. Together with guest personalities that specialize in Open Source, DevOps, Security and Enterprise Architecture, this book also presents discourse-formatted philosophical discussions and open-source sustainability viewpoints from which your next-gen apps will profit - and your users will benefit.
-
-In this book you will follow the authors in the iterative evolution of a real project from conception to distribution - all with commentary, complete code resources, built, and packaged Native Apps for reference and staged Capture the Flag (CTF) challenges that progress in difficulty as your comprehension of the system grows.
-
-### About the Topic
+:::info Progress Update
 
-Tauri is a brand new way to make cross-platform native-apps for web, desktop and mobile. At this very moment, the pre-alpha version of this MIT licensed community-based software is being prepared for public release: https://github.com/tauri-apps/tauri
+We're actively working on authoring and writing The Tauri Book. We've encountered delays due to the immense growth of Tauri but have recently re-prioritized this project. While we don't yet have details on the timelines of a release, you can keep an eye on this page for updates.
 
-Tauri introduces novel methods for WebView integration and innovative patterns for robust threat evasion. The 1.0 release will ship with a multipurpose white-box analyzer and decompiler for any kind of binary and an integrated CLI for ingesting any type of HTML; which, when combined, provides developers and security teams with a holistic platform that has never existed as a single unit before.
+We'd like to apologize for this being delayed beyond the originally communicated publish date. If you've donated through GitHub Sponsors or Open Collective and would like to request a refund you may do so via Open Collective: [Contact Tauri on Open Collective](https://opencollective.com/tauri/contact).
+:::
 
-Tauri bridges communities and opens up new opportunities for everyone from the front end developer all the way to the low-level security and network administrators. Due to this level of complexity and robustness, it is important to publish a reference guide that will necessarily be updated as major versions are released.
+### Overview
 
-### What you will learn
+The Tauri Book will guide you through the history of Tauri and the design decisions we've made. It will also help you become a better developer by looking beyond the basics and understand why decisions around privacy, security and sustainability are important.
```

Comment:
```suggestion
The Tauri Book will guide you through the history of Tauri and the design decisions we've made. It will also talk in depth about why privacy, security and sustainability are important. Fundamental discussions you can apply to any modern software project.
```

Help you become a better developer, is imo not a great message. Tried to rephrase it.

Reply:
Not sure if there's a way to suggest on a suggestion, so edited your comment slightly to make it a full sentence.